packages/amplify-cli/src/extensions/amplify-helpers/push-resources.ts

```diff
@@ -114,3 +118,24 @@ export async function storeCurrentCloudBackend(context: $TSContext) {
 
   await Promise.all(providerPromises);
 }
+
+const getResources = async (context: $TSContext): Promise<IAmplifyResource[]> => {
```

Comment:
This function is a dup of this - https://github.com/aws-amplify/amplify-cli/pull/8228/files#diff-3f6f75efd01ee3b6a12aadd99f3f7afe7a7c1e3164364a6d1e7a68da1b09fc7fR37 
move to a common function